python/pysmurf/client/util/smurf_util.py

```diff
@@ -2415,10 +2415,42 @@ def get_hemt_drain_current(self):
 
         # on cryostat card
         hemt_Vd_series_resistor=200  #Ohm
-        hemt_Id_mA=2.*1000.*(self.get_cryo_card_hemt_bias())/hemt_Vd_series_resistor - self._hemt_Id_offset
+        hemt_Id_mA=2.*1000.*self.get_cryo_card_hemt_bias()/
+            hemt_Vd_series_resistor - self._hemt_Id_offset
 
         return hemt_Id_mA
 
+
+    def set_50k_enable(self, write_log=False):
```

Comment:
Need confirm from Joe that we have actual read access to cryostat card, not just memory of previous commanding.

Reply:
Joe confirmed, resolving.